react-components/src/hooks/useCameraNavigation.tsx

```diff
@@ -56,9 +57,14 @@ export const useCameraNavigation = (): CameraNavigationActions => {
     await fitCameraToModelNode(modelMappings.revisionId, nodeId.id);
   };
 
+  const fitCameraToState = (cameraState: CameraState): void => {
+    viewer.cameraManager.setCameraState(cameraState);
```

Comment:
I see in the Toolbar story check for the `cameraState` validation but it make more sense to check within `fitCameraToState`

Reply:
Why? I don't think it makes sense to accept undefined here? By that logic every argument to every function should be `T | undefined`.